README.adoc

```diff
@@ -66,7 +66,7 @@ sense. With SDKMAN!, the `~/.m2/mvnd.properties` file is typically not needed at
 brew install mvndaemon/homebrew-mvnd/mvnd
 ----
 
-Note: There are two formulae: the `mvnd` that install latest, and `mvnd@1` that installs 1.x line.
+Note: There are two formulae. Contrary to the recommendation by the [homebrew cask standards](https://docs.brew.sh/Acceptable-Casks#beta-unstable-development-nightly-or-legacy), the `mvnd` installs the preview 2.x version , and `mvnd@1` installs the stable 1.x version.
```

Comment:
Remove "Contrary to the recommendation by the [homebrew cask standards](https://docs.brew.sh/Acceptable-Casks#beta-unstable-development-nightly-or-legacy)" as it's not relevant to the target audience, though you can file a bug for that if you want to reverse this. 

the `mvnd` --> `mvnd`